signup.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Linkbrary</title>
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css" />
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="sign.css">
+</head>
+<body>
+  <div class="sign-wrap">
+    <div class="sign-top">
+      <a href="/" class="sign-top_logo">
+        <img src="imgs/sign/logo.png" alt="로고">
+      </a>
+      <div class="sign-top_question">
+        <p>이미 회원이신가요?</p>
+        <a href="/signin" class="sign-question_join">로그인 하기</a>
+      </div>
+    </div>
+    <form class="sing-middle">
+      <div class="sing-userinfo">
+        <div class="sign-userinput">
+          <label for="useremail">이메일</label>
+          <input type="text" name="useremail" id="useremail" placeholder="codeit@codeit.com">
```

Comment:
input 태그 value에 해당하는 값을 placeholder에 넣어주신 것으로 보여요. 

placeholder attribute의 용처를 다시 살펴보고 변경해보면 좋을 것 같아요.
예를 들어, 이메일의 placeholder 예시는 `"e.g. codeit@codeit.com"`이, 패스워드의 placeholder 예시는 `"영문 + 숫자 조합 8자리 이상"` 같은 것이 들어갈 수 있을 것 같아요.

> placeholder attribute provides a brief hint to the user as to what kind of information is expected in the field. It should be a word or short phrase that provides a hint as to the expected type of data, rather than an explanation or prompt

- 참고: [MDN 문서](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder)